packages/legend-application-studio/src/stores/graphModifier/StoreRelational_GraphModifierHelper.ts

```diff
@@ -421,3 +429,62 @@ export const rootRelationalSetImp_setPropertyMappings = action(
     );
   },
 );
+
+// --------------------------------------------- Post-Processor -------------------------------------
+
+export const relationalDatabaseConnection_addPostProcessor = action(
+  (
+    connectionValueState: RelationalDatabaseConnectionValueState,
+    postProcessor: PostProcessor,
+    observerContext: ObserverContext,
+  ): void => {
+    addUniqueEntry(
+      connectionValueState.connection.postProcessors,
+      observe_PostProcessor(postProcessor, observerContext),
+    );
+  },
+);
+
+export const relationalDatabaseConnection_deletePostProcessor = action(
+  (
+    connectionValueState: RelationalDatabaseConnectionValueState,
+    postProcessor: PostProcessor,
+  ): void => {
+    deleteEntry(connectionValueState.connection.postProcessors, postProcessor);
+  },
+);
+
+export const mapperPostProcessor_addMapper = action(
+  (postProcessor: PostProcessor, mapper: Mapper): void => {
```

Comment:
the target here is `mapperPostProcessor: MapperPostProcessor`, not `postProcessor`. Avoid casting like below, that's basically lying to Typescript, which could be dangerous.

Reply:
mm i see!